### DISCUSSION TITLE
How to "strikethrough"?

### DISCUSSION THREAD

**Discussion by glyh** [kovidgoyal/kitty#4632] [Q&A] [answered]
Here's how it works on alacritty:
![image](https://user-images.githubusercontent.com/12870742/152491706-2638d7c2-e5cc-474d-84cf-4fa0ddb90d59.png)
And here's how it works on kitty:
![image](https://user-images.githubusercontent.com/12870742/152491722-2960ac51-eadb-4eca-b74e-a748c2906eb2.png)
Is there a way to get the behavior like in alacritty?

Thanks a lot!

**Comment by kovidgoyal** [comment, marked as answer]
https://github.com/kovidgoyal/kitty/commit/37d848372864a70e4912f39f7ef75c0060069a0a

**Reply by glyh** [reply]
Thanks for the fix!

**Comment by neingeist** [comment]
To anyone who ends up here via Google:

- kitty-0.26.5  (at least the Fedora built RPM) also did not render it correctly
- kitty-0.27.1 is fine
